packages/okta-react/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 3.0.7
+
+### Bug Fixes
+
+- [#903](https://github.com/okta/okta-oidc-js/pull/903) fixes SecureRoute to run logic only if route matches
```

Comment:
Could we phrase this to match the problems they see?

"Fixes SecureRoute to not require authentication unless the route actually matches"

It's small, but if they are focused on their problem, they may not notice your original as relevant.